resources/definitions/creality_cr100.def

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "Creality CR-100",
+    "version": 4,
```

Comment:
The tests indicate (correctly) that this version number is incorrect. It's a version number to indicate the format of this file. The current version is 2. But really, it's best to just leave this out and let it inherit from `creality_base`.
```suggestion
```

Reply:
so do i delete that line?

Reply:
That is my suggested change, yes.

Reply:
done